src/ert/_c_wrappers/job_queue/ert_script.py

```diff
@@ -83,26 +75,25 @@ def initializeAndRun(
         try:
             return self.run(*arguments)
         except AttributeError as e:
+            error_msg = e
             if not hasattr(self, "run"):
-                self.__failed = True
-                return (
-                    f"Script '{self.__class__.__name__}' "
-                    "has not implemented a 'run' function"
-                )
-            self.outputStackTrace(e)
+                error_msg = "No 'run' function implemented"
```

Comment:
No need / necessary to provide the name anymore as in the old code?

Reply:
The call of `output_stack_trace` will add the name of the class, there is no need to have it twice in the same error message